src/transport/MixTransport.cpp

```diff
@@ -112,6 +113,33 @@ void MixTransport::getSpeciesFluxes(size_t ndim, const doublereal* const grad_T,
             sum[n] += fluxes[n*ldf + k];
         }
     }
+
+	//Modification: amibipolar diffusion. Ref.- Prager et al. 2007
+	//some dummy variables
+	doublereal* beta;
```

Comment:
This variable is used without every being allocated. If you need a vector of size `m_nsp`, you should use `vector_fp`.